packages/app/src/app/components/dashboard/InputSelect.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { SelectHTMLAttributes } from 'react';
+import styled from 'styled-components';
+import { Icon, Stack } from '@codesandbox/components';
+import { Label } from './Label';
+
+const RelativeWrapper = styled.div`
```

Comment:
food for thought: in v2 we use a lot of instances of `Box` for quick styling for a div. Should we add it here as well as a copy of the `Element` component for future compatibility?. We can limit the usage of `styled.div` for a single css rule that we need in the hierarchy this way.

Reply:
Yeah that's a good idea. We can use `<Element css={{}} />`.